cupy/linalg/solve.py

```diff
@@ -274,10 +275,20 @@ def inv(a):
         getrf = cusolver.sgetrf
         getrf_bufferSize = cusolver.sgetrf_bufferSize
         getrs = cusolver.sgetrs
-    else:  # dtype == 'd'
+    elif dtype == 'd':
         getrf = cusolver.dgetrf
         getrf_bufferSize = cusolver.dgetrf_bufferSize
         getrs = cusolver.dgetrs
+    elif dtype == 'F':
+        getrf = cusolver.cgetrf
+        getrf_bufferSize = cusolver.cgetrf_bufferSize
+        getrs = cusolver.cgetrs
+    elif dtype == 'D':
+        getrf = cusolver.zgetrf
+        getrf_bufferSize = cusolver.zgetrf_bufferSize
+        getrs = cusolver.zgetrs
+    else:
+        raise RuntimeError("unsupported dtype")
```

Comment:
How about a `ValueError` (e.g. similar to https://github.com/cupy/cupy/blob/master/cupy/core/core.pyx#L1932) and a nit but let's stick to single quotes for consistency.
```suggestion
        raise ValueError('unsupported dtype')
```